src/FunctionBlocks/LogicFunctionBlock.cpp

```diff
@@ -2,7 +2,7 @@
 #include "knxprod.h"
 
 LogicFunctionBlock::LogicFunctionBlock(uint8_t channelIndex, LogicFunctionBlockType type)
-    : FunctionBlock(channelIndex, type == LogicFunctionBlockType::LogicOR ? "OR" : "AND"),
+    : FunctionBlock(channelIndex, type == LogicFunctionBlockType::LogicOR ? "OR" : (type == LogicFunctionBlockType::LogicAND ? "AND" : "COUNT")),
```

Comment:
Das sollten wir nun in eine statische Hilfsfunktion auslagern, vielleicht kommen da ja noch mehr dazu

Reply:
Ja, diese Form skaliert nicht gut mit größerer Anzahl von (Sub)Typen. Wird beim Blocktyp Aggregation mit 5 Typen noch deutlicher. Hatte es dort erst einmal in ein Makro ausgelagert für bessere Lesbarkeit als in nur einer Zeile. Siehe https://github.com/OpenKNX/OFM-FunctionBlocks/pull/3/files#diff-21a11a6bb4060f3d236ba08e39da2f0541da66dde2f07820c1be33a038e34f95R4-R15

Wir sollten da vor allem eine Lösung finden die möglichst einheitlich ist über alle Funktionsblöcke.